apps/website/src/pages/docs/_layout.astro

```diff
@@ -16,7 +17,8 @@ const currentPage = new URL(Astro.request.url).pathname;
     <div class='content-wrapper' id='article'>
       <h1 id='overview'>{content.title}</h1>
       <slot />
-      <div class='bottom-links'>
+      <div class='aftermain'>
+        {content.related ? <RelatedComponents related={content.related} /> : null}
```

Comment:
non-blocking: Just a thought. Would it be good to keep the related components section near the top of the page, just after the first code example? Users may find it easy to see all related components at once before deciding which component from that category best suits their needs.

Reply:
no, this is "extra" content. the documentation for the current component should come first

for deciding which component, i think we should have text explanation, similar to how we mention **toast** in the intro section of **alert** page